projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -377,37 +351,31 @@ public static void main(String[] args) {
     // Paths to the CSVs files
     final Path agencyTxt = Paths.get(args[0]);
     final Path calendarTxt = Paths.get(args[1]);
-    final Path fareAttributesTxt = Paths.get(args[2]);
-    final Path fareRulesTxt = Paths.get(args[3]);
-    final Path routesTxt = Paths.get(args[4]);
-    final Path shapesTxt = Paths.get(args[5]);
-    final Path stopTimesTxt = Paths.get(args[6]);
-    final Path stopsTxt = Paths.get(args[7]);
-    final Path translationsTxt = Paths.get(args[8]);
-    final Path tripsTxt = Paths.get(args[9]);
+    final Path routesTxt = Paths.get(args[2]);
+    final Path stopTimesTxt = Paths.get(args[3]);
+    final Path stopsTxt = Paths.get(args[4]);
+    final Path translationsTxt = Paths.get(args[5]);
+    final Path tripsTxt = Paths.get(args[6]);
 
     // Paths to the output files
-    final String agencyOutputPath = args[args.length - 10];
-    final String calendarOutputPath = args[args.length - 9];
-    final String fareAttributesOutputPath = args[args.length - 8];
-    final String fareRulesOutputPath = args[args.length - 7];
-    final String routesOutputPath = args[args.length - 6];
-    final String shapesOutputPath = args[args.length - 5];
+    final String agencyOutputPath = args[args.length - 7];
+    final String calendarOutputPath = args[args.length - 6];
+    final String routesOutputPath = args[args.length - 5];
     final String stopTimesOutputPath = args[args.length - 4];
     final String stopsOutputPath = args[args.length - 3];
     final String translationsOutputPath = args[args.length - 2];
     final String tripsOutputPath = args[args.length - 1];
 
     saveAgency(agencyTxt, agencyOutputPath);
     saveCalendar(calendarTxt, calendarOutputPath);
-    saveFareAttributes(fareAttributesTxt, fareAttributesOutputPath);
-    saveFareRules(fareRulesTxt, fareRulesOutputPath);
     saveRoutes(routesTxt, routesOutputPath);
-    saveShapes(shapesTxt, shapesOutputPath);
     saveStopTimes(stopTimesTxt, stopTimesOutputPath);
     saveStops(stopsTxt, stopsOutputPath);
     saveTranslations(translationsTxt, translationsOutputPath);
     saveTrips(tripsTxt, tripsOutputPath);
+    // TODO: remove later
+    testCalendarOnlyTodayService(calendarTxt);
```

Comment:
Here's some examples for tests:
https://github.com/google/startup-os/tree/master/common/tests

If you add a similar test, it will be the first time in `hasadna` repo, so you might need to add some dependencies. If you wanna do that, let us know and we will guide you.